networkx/linalg/tests/test_algebraic_connectivity.py

```diff
@@ -11,17 +11,28 @@
 import networkx as nx
 from networkx.testing import almost_equal
 
-try:
-    from scikits.sparse.cholmod import cholesky
+methods = ("tracemin_pcg", "tracemin_lu", "lanczos", "lobpcg")
 
-    _cholesky = cholesky
-except ImportError:
-    _cholesky = None
 
-if _cholesky is None:
-    methods = ("tracemin_pcg", "tracemin_lu", "lanczos", "lobpcg")
-else:
-    methods = ("tracemin_pcg", "tracemin_chol", "tracemin_lu", "lanczos", "lobpcg")
+def test_algebraic_connectivity_tracemin_chol():
+    """Test that "tracemin_chol" raises an exception."""
+    with pytest.raises(nx.NetworkXError):
+        G = nx.barbell_graph(5, 4)
+        nx.algebraic_connectivity(G, method="tracemin_chol")
```

Comment:
```suggestion
    G = nx.barbell_graph(5, 4)
    with pytest.raises(nx.NetworkXError):
        nx.algebraic_connectivity(G, method="tracemin_chol")
```
One nit-pick to avoid the unlikely case that the test passes due to something goeing wrong in the graph creation rather than the connectivity. Ditto for the two following tests.